docs/src/dataDisplay/list/list.jsx

```diff
@@ -121,28 +121,29 @@ export default class ElementsList extends React.Component {
                     </List>
 
                     {/* Horizontal */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         Horizontal
-                        <Header.Subheader>
-                            A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A basic card has a faint border wrapping it's contents. It's box model is fluid so that it can adapt to it's parent container's width.
```

Comment:
"it's" => "its" here as well.
Actually ... was this just copied and pasted from above...?  Is it even relevant here?  Copy-and-paste error?  Same goes for several additional placements below.

Reply:
Fixed.

Reply:
Looks to have been removed.  Resolving.